Dolapo47/Auto-Mart
# create route to retrieve all available cars in the app [finish #166152124)]

#### What does this PR do?
- A summary of what this task is about
This PR is designed to retrieve all available cars in the app

#### Description of Task to be completed?
- A list of the major tasks you did to complete this feature/bug/chore
create the get all car route
create a function to filter cars that are available
display all available cars to the users

#### How should this be manually tested?
- Outlined steps on how someone else can successfully test this feature/chore/bug to ensure it works as expected
navigate to GET/http://localhost:3000/api/v1/car on postman and send

#### What are the relevant pivotal tracker stories?
- PT Story link - (format - `[#166152124](https://www.pivotaltracker.com/n/projects/2346179)`)
